PostHog/Classes/PHGPayloadManager.m

```diff
@@ -102,15 +116,11 @@ - (void)identify:(NSString *)distinctId properties:(NSDictionary *)properties op
 {
     NSCAssert1(distinctId.length > 0, @"distinctId (%@) must not be empty.", distinctId);
 
-    NSString *anonymousId = [options objectForKey:@"$anon_distinct_id"];
-    if (anonymousId) {
-        [self saveAnonymousId:anonymousId];
-    } else {
-        anonymousId = self.cachedAnonymousId;
-    }
+    [self saveAnonymousId:self.integration.distinctId];
+    [self.integration saveDistinctId:distinctId];
 
     PHGIdentifyPayload *payload = [[PHGIdentifyPayload alloc] initWithDistinctId:distinctId
-                                                                     anonymousId:anonymousId
+                                                                     anonymousId:self.cachedAnonymousId
```

Comment:
What is the context behind this? 
`Adjusts handling of distinct/anonymous ids to match posthog-js` = ?

Reply:
The library right now isn't saving "current" distinct_id and the "previous" distinct_id in a way that allows for implicit handling in functions such as isFeatureEnabled. The client version of these functions don't need to be explicitly passed a distinct_id because theyll use the cache'd one to determine the flag